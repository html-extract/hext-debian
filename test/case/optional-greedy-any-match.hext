<?+* @inner-html:x @inner-html==""! />
