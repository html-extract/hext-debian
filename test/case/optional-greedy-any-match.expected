{"x":"<head>\n    <meta charset=\"utf-8\"/>\n    <title>Example</title>\n  </head>\n  <body>\n    <a href=\"one.html\">  Page 1</a>\n    <a href=\"two.html\">  Page 2</a>\n    <a href=\"three.html\">Page 3</a>\n  \n</body>"}
{"x":["\n    <meta charset=\"utf-8\"/>\n    <title>Example</title>\n  ","\n    <a href=\"one.html\">  Page 1</a>\n    <a href=\"two.html\">  Page 2</a>\n    <a href=\"three.html\">Page 3</a>\n  \n"]}
{"x":"Example"}
{"x":["  Page 1","  Page 2","Page 3"]}
